src/common/options/rgw.yaml.in

```diff
@@ -406,6 +406,10 @@ options:
 - name: rgw_lc_debug_interval
   type: int
   level: dev
+  desc: Lifecycle processing interval
+  long_desc: When set to greater than 0, treat each this many seconds as a day.
```

Comment:
I'm not sure I agree with you on your police work there, Lou.

"treat each this many seconds as a day" reads like something from Zelda 1.